LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright © 2016 Catalin Luntraru
```

Comment:
Yeah, I really like her templates when I don't want to hassle with setting up build tasks and what not :) I'll link to her site and the original repository.